Tasks/Common/VstsAzureHelpers_/Utility.ps1

```diff
@@ -493,26 +493,11 @@ function Disconnect-AzureAndClearContext {
 
     try {
         if ($authScheme -eq 'ServicePrincipal') {
-            Write-Verbose "Trying to disconnect from Azure and clear context"
-
-            if (Get-Command -Name "Disconnect-AzureRmAccount" -ErrorAction "SilentlyContinue") {
```

Comment:
Instead of removing this completely, can you do disconnect as well at process scope wherever it is scope option is supported.